src/Features/ERDDAP/hooks/tabledap.ts

```diff
@@ -0,0 +1,76 @@
+import * as Sentry from "@sentry/react"
+import { useQuery, QueryResult } from "react-query"
+
+import { resultToTimeseries, tabledapUrl } from "Shared/erddap"
+import { ErddapJson } from "Shared/erddap/types"
+import { proxytizeUrl } from "Shared/proxyUrl"
+import { DataTimeSeries } from "Shared/timeSeries"
+
+import { PlatformTimeSeries } from "../types"
+import { defaultQueryConfig } from "./hookConfig"
+
+const getDataset = (timeSeries: PlatformTimeSeries, startTime?: Date) => {
+  return async () => {
+    const url = urlBuilder(timeSeries, startTime)
```

Comment:
Codacy found an issue: ['urlBuilder' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)